pkg/sentry/arch/arch_aarch64.go

```diff
@@ -32,26 +32,32 @@ import (
 const (
 	// SyscallWidth is the width of insturctions.
 	SyscallWidth = 4
+
+	// fpsimd_magic is the magic number which is used in fpsimd_context.
+	fpsimd_magic = 0x46508001
+
+	// fpsimd_context_size is the size of fpsimd_context.
+	fpsimd_context_size = 0x210
```

Comment:
Please name these in the standardCamelCase form.

Reply:
Done.